packages/front-end/components/Metrics/MetricToolTipBody.module.scss

```diff
@@ -0,0 +1,22 @@
+@import "../../styles/colors.scss";
+
+.description {
+  position: relative;
+  height: 100px;
```

Comment:
Change this to `max-height` so that short descriptions don't have so much whitespace at the bottom.

You'll also probably need to change the `:after` styles a bit - changing `bottom: 0` to `top: 40px` should be good enough.